ruby/tests/repeated_field_test.rb

```diff
@@ -20,6 +20,7 @@ def test_acts_like_an_array
       :iter_for_each_with_index, :dimensions, :copy_data, :copy_data_simple,
       :nitems, :iter_for_reverse_each, :indexes, :append, :prepend]
     arr_methods -= [:union, :difference, :filter!]
+    arr_methods -= [:intersection, :deconstruct] # ruby 2.7 methods we can ignore
```

Comment:
Not 💯 sure we can just ignore these methods, either way the segfault is more concerning at the moment